google-cloud-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -117,7 +117,8 @@
           + "  \"project_id\": \"someprojectid\",\n"
           + "  \"client_email\": \"someclientid@developer.gserviceaccount.com\",\n"
           + "  \"client_id\": \"someclientid.apps.googleusercontent.com\",\n"
-          + "  \"type\": \"service_account\"\n"
+          + "  \"type\": \"service_account\",\n"
+          + "  \"quota_project_id\": \"some-quota-project-id\"\n"
```

Comment:
Can we separate this out into a second example service account json and add a test for when it's not present?

Reply:
will do!